packages/fiori-elements-writer/test/__snapshots__/lrop.test.ts.snap

```diff
@@ -14019,7 +14019,7 @@ EntityType=\\"com.sap.gateway.srvd.dmo.sd_travel_mduu.v0001.TravelAgencyType\\">
           \\"options\\": {
             \\"settings\\": {
               \\"editableHeaderContent\\": false,
-              \\"contextPath\\": \\"/Booking\\"
+              \\"contextPath\\": \\"/Travel/Booking\\"
```

Comment:
@IainSAP That's not perfect, I guess: it should be the navigation property key, not the entitySet that the navigation points to: 
<img width="691" alt="image" src="https://github.com/SAP/open-ux-tools/assets/126068396/da18a28f-e0f3-4cf5-b660-c34f34d2aaa4">